src/main/app/src/app/shared/material-form-builder/form-components/documenten-lijst/documenten-lijst.component.ts

```diff
@@ -44,7 +44,7 @@ export class DocumentenLijstComponent extends FormComponent implements OnInit {
                 document.creatiedatum = this.datumPipe.transform(document.creatiedatum); // nodig voor zoeken
                 document['viewLink'] = `/informatie-objecten/${document.uuid}`;
                 document['downloadLink'] = this.informatieObjectenService.getDownloadURL(document.uuid);
-                if (this.data.documentenCheckedVoorOndertekenen?.includes(document.uuid)) {
+                if (this.data.documentenCheckedVoorOndertekenen?.includes(document.uuid) || document.ondertekening) {
```

Comment:
Dit begrijp ik niet. Ga je het hier nu mogelijk maken om een document wat al ondertekend is weer niet ondertekend te maken of zo?

Reply:
Nee, een ondertekend document kan niet meer uitgevinkt worden op deze pagina want de checkbox is disabled. Dit zorgt er echter wel voor dat de checkbox bij een ondertekend document altijd checked is.